docs/en/ingest-management/release-notes/release-notes-8.12.asciidoc

```diff
@@ -230,6 +230,26 @@ sudo systemctl start elastic-agent
 
 ====
 
+[[known-issue-37754]]
+.Performance regression in AWS S3 inputs using SQS notification
+[%collapsible]
+====
+
+*Details*
+
+In 8.12 the default memory queue flush interval was raised from 1 second to 10 seconds. In many configurations this improves performance because it allows the output to batch more events per round trip, which improves efficiency. However, the SQS input has an extra bottleneck that interacts badly with the new value.
+
+For more details see {beats-issue}37754[#37754].
+
+*Impact* +
+
+If you are using the Elasticsearch output, and your configuration uses a performance preset, switch it to `preset: latency`. If you use no preset or use `preset: custom`, then set `queue.mem.flush.timeout: 1` in your output configuration.
```

Comment:
@cmacknz, should we set the `queue.mem.flush.timeout` to `1` or `1s`? Does this make a difference if the output parses it as a `time.Duration`?

The example values in the [docs](https://www.elastic.co/guide/en/beats/filebeat/8.12/configuring-internal-queue.html) are in the `1s` format.

Reply:
Thanks, updating this in https://github.com/elastic/ingest-docs/pull/871

Will do the same for Beats.